### ISSUE TITLE
RFE: Persistent Skirmish/multiplayer game options and ability to restart with said options

### ISSUE THREAD

**Issue by MagicFab** [OpenRA/OpenRA#10533]
I'd like to be able to restart a game when in Skirmish or multiplayer modes while kepeing the same game options.

When initially setting up such a game, one may alter game speed, money available, player slots (including AI, location, teams, etc), map, etc. Once the game is finished the only choice is to completely exit and having to set again all options manually to restart the same type of game.

It would be convenient to be able to restart the same game with the same options quickly (add a "Play again" option?), while keeping the option to reset completely and start over.
##

**Comment by whinis** [comment]
Now that #10264 has been merged I am starting work on just this, skirmish is likely to happen before multiplayer.

**Comment by abcdefg30** [comment]
Closing as fixed/duplicate of #4122.
